win32ss/user/user32/windows/input.c

```diff
@@ -657,11 +662,19 @@ IntLoadKeyboardLayout(
     UNICODE_STRING ustrKLID;
     WCHAR wszRegKey[256] = L"SYSTEM\\CurrentControlSet\\Control\\Keyboard Layouts\\";
     WCHAR wszLayoutId[10], wszNewKLID[10], szImeFileName[80];
```

Comment:
Why the two first buffers are of size 10 ? Should they be instead of size `KL_NAMELENGTH` ? (== 9 , i.e. representing 8 hex characters and one terminating NULL)

Reply:
Fixed.